Helper/Cart.php

```diff
@@ -61,7 +61,7 @@
 use Bolt\Boltpay\Model\ErrorResponse as BoltErrorResponse;
 use Bolt\Boltpay\Helper\MetricsClient;
 use Bolt\Boltpay\Helper\FeatureSwitch\Decider as DeciderHelper;
-
+use Magento\Framework\Serialize\Serializer\Serialize;
```

Comment:
Not availvable before Magento 2.2. We may not officially support older Magento versions, but may still have merchant stores running on them.

@matt-thomason @njfaries @vitaliyreznikov how do we proceed here?